libs/ha/ha_common_libs_k8s.py

```diff
@@ -778,18 +779,19 @@ def start_random_mpu(self, event, s3_data, bucket_name, object_name, file_size,
                 parts_etag.append({"PartNumber": i, "ETag": p_tag["ETag"]})
                 LOGGER.info("Uploaded part %s", i)
             except CTException as error:
-                LOGGER.error("Error: %s", error)
+                LOGGER.exception("Error: %s", error)
                 if event.is_set():
                     exp_failed_parts.append(i)
                 else:
                     failed_parts.append(i)
-                LOGGER.info("Failed to upload part %s", i)
+                LOGGER.exception("Failed to upload part %s", i)
```

Comment:
No need to print traceback again. This can be LOGGER.info only as traceback will be printed at line 782.